### PR TITLE
Doc/4th improvement

### PR DESCRIPTION
**Pull request by HansRobo**
## Types of PR

- [x] Document 

## Description

- Allow custom words with SpellCheck CI
- Fix typos
- Update content
  - UserDefinedValueCondition 
  - Simple Sensor Simulator
  - OpenSCENARIO support
    - Add new contains of ver1.2 
- Add pages
  - Communication
    - Topic list
- Arrange directories or files
  - Delete `etc` directory
  - Merge `HowToContribute` and `CONTRIBUTING`
  - Delete `doc/LICENSE`
  - Delete `CONTACT`
    - Put links to issue and pr page on the footer instead 
- Change layout
  - Make the content width more wide

### CODE REVIEW

File: docs/ReleaseNotes.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
...
 
-|                      Feature                       |                                            Brief summary                                             |      Category       |                          Pull request                           |                  Contributor                  |
-| -------------------------------------------------- | ---------------------------------------------------------------------------------------------------- | ------------------- | --------------------------------------------------------------- | --------------------------------------------- |
-| Add `API::getRelativePose()` argument for function | Add `API::getRelativePose()` argument for getting relative distance between lanalet pose and entity. | `traffic_simulator` | [#809](https://github.com/tier4/scenario_simulator_v2/pull/809) | [hakuturu583](https://github.com/hakuturu583) |
+|                      Feature                       | Brief summary                                                                                        |      Category       |                          Pull request                           |                  Contributor                  |
```
Comment (hakuturu583) [inline docs/ReleaseNotes.md:8]: Please align to the center.

File: docs/developer_guide/SimpleSensorSimulator.md [hunk 2]
```diff
@@ -2,25 +2,98 @@
...
+With this simulation, we can get lidar point-cloud data based on simple ray-casting algorithm.
+
+### Interfaces
+
+| interface                                   | type                                   | note                                                                                       |
+|---------------------------------------------|----------------------------------------|--------------------------------------------------------------------------------------------|
+| `traffic_simulator::API::attachLidarSensor` | C++ traffic simulator API interface    |                                                                                            |
+| `attach_lidar_sensor`                       | ZeroMQ traffic simulator API interface | See [ZeroMQ Interfaces documentation](/docs/developer_guide/ZeroMQ.md)<br/>TCP Port : 5563 |  
+
+### Configuration
```
Comment (hakuturu583) [inline docs/developer_guide/SimpleSensorSimulator.md:26]: Please remove it.
Configuration should be auto-generated from protobuf-gen-c-doc

File: docs/developer_guide/SimpleSensorSimulator.md [hunk 3]
```diff
@@ -2,25 +2,98 @@
...
+
+| grid type      | value | description                                                         |
+|----------------|-------|---------------------------------------------------------------------|
+| occupied grid  | 100   | a grid determined by a simulated lidar sensor that an object exists |
+| empty grid     | 0     | an empty grid proved by a simulated ray-cast passing through        |    
+| invisible grid | 50    | a grid that is out of range or occlusion of simulated lidar sensor  |
+
+
+### Configuration
+| name              | type   | unit   | description                                                                                                                         |
```
Comment (hakuturu583) [inline docs/developer_guide/SimpleSensorSimulator.md:67]: Please remove this table.
Configuration should be auto-generated from protobuf-gen-c-doc

File: docs/developer_guide/SimpleSensorSimulator.md [hunk 4]
```diff
@@ -2,25 +2,98 @@
...
+### Interfaces
+
+| interface                                   | type                                   | note                                                                                       |
+|---------------------------------------------|----------------------------------------|--------------------------------------------------------------------------------------------|
+| `traffic_simulator::API::attachDetectionSensor` | C++ traffic simulator API interface    |                                                                                            |
+| `attach_detection_sensor`                       | ZeroMQ traffic simulator API interface | See [ZeroMQ Interfaces documentation](/docs/developer_guide/ZeroMQ.md)<br/>TCP Port : 5564 |  
+
+
+### Configuration
+
```
Comment (hakuturu583) [inline docs/developer_guide/SimpleSensorSimulator.md:92]: Please remove this table.
Configuration should be auto-generated from protobuf-gen-c-doc

**Review by hakuturu583 — CHANGES_REQUESTED** [review]

File: docs/developer_guide/SimpleSensorSimulator.md (hunk 2 above)
Comment (HansRobo) [reply docs/developer_guide/SimpleSensorSimulator.md:26]: OK. I'll replace it with the link to [auto-generated document](https://tier4.github.io/scenario_simulator_v2-docs/proto_doc/protobuf/) 
But, I think the unit information of each item in the configuration is the minimum required.
So, please check unit colomn!
After that, I'll add the unit information to `.proto` files. 
(This means unit information will be added to auto-generated documents)

File: docs/developer_guide/SimpleSensorSimulator.md [hunk 5]
```diff
@@ -2,25 +2,98 @@
...
+| interface                                   | type                                   | note                                                                                       |
+|---------------------------------------------|----------------------------------------|--------------------------------------------------------------------------------------------|
+| `traffic_simulator::API::attachOccupancyGridSensor` | C++ traffic simulator API interface    |                                                                                            |
+| `attach_occupancy_grid_sensor`                       | ZeroMQ traffic simulator API interface | See [ZeroMQ Interfaces documentation](/docs/developer_guide/ZeroMQ.md)<br/>TCP Port : 5565 |  
+
+### Occupancy Grid Values
+
+| grid type      | value | description                                                         |
+|----------------|-------|---------------------------------------------------------------------|
+| occupied grid  | 100   | a grid determined by a simulated lidar sensor that an object exists |
```
Comment (hakuturu583) [inline docs/developer_guide/SimpleSensorSimulator.md:61]: Can you sort by value?

**Review by hakuturu583 — CHANGES_REQUESTED** [review]

File: docs/developer_guide/SimpleSensorSimulator.md [hunk 6]
```diff
@@ -2,25 +2,79 @@
...
+### Interfaces
+
+| interface                                   | type                                   | note                                                                                       |
+|---------------------------------------------|----------------------------------------|--------------------------------------------------------------------------------------------|
+| `traffic_simulator::API::attachLidarSensor` | C++ traffic simulator API interface    |                                                                                            |
+| `attach_lidar_sensor`                       | ZeroMQ traffic simulator API interface | See [ZeroMQ Interfaces documentation](/docs/developer_guide/ZeroMQ.md)<br/>TCP Port : 5563 |  
+
+### Configuration
+
+See [the LidarConfigulation documentation](https://tier4.github.io/scenario_simulator_v2-docs/proto_doc/protobuf/#lidarconfiguration)
```
Comment (hakuturu583) [inline docs/developer_guide/SimpleSensorSimulator.md:28]: Typo detected.

Comment (hakuturu583) [reply docs/developer_guide/SimpleSensorSimulator.md:28]: Configuration?

**Review by hakuturu583 — CHANGES_REQUESTED** [review]

**Review by hakuturu583 — APPROVED** [review]

**Pull request merged**